### DISCUSSION TITLE
Unable to access environmental variables in the client

### DISCUSSION THREAD

**Discussion by Onibenjo** [vercel/next.js#12889] [Help] [answered]
Upgrading to next 9.4.0, I had to remove all environmental variables in `next.config.js`.
But I am unable to access the env variable on the client.
Console logging `process.env` logs the variables on the console but logs undefined on the browser

**Comment by sambhav-gore** [comment]
How are you naming the variables? as per the docs if you want to access in client side then it must begin with `NEXT_PUBLIC_`

**Reply by Onibenjo** [reply]
I did that

**Comment by cdock1029** [comment, marked as answer]
Remove `dotenv` if you had been using it

https://github.com/zeit/next.js/commit/5da1dcf669cff474eb6521008173158a5d8f6ebe

**Reply by Onibenjo** [reply]
I removed every code initializing dotenv

**Reply by cdock1029** [reply]
And the dependency itself?

https://github.com/zeit/next.js/pull/12859

**Reply by Onibenjo** [reply]
I haven't removed the dependency

**Reply by Onibenjo** [reply]
I will try it now

**Reply by Onibenjo** [reply]
that worked
